dev/integration_tests/flutter_gallery/lib/demo/contacts_demo.dart

```diff
@@ -294,7 +294,7 @@ class ContactsDemoState extends State<ContactsDemo> {
                       icon: Icons.map,
                       tooltip: 'Open map',
                       onPressed: () {
-                        _scaffoldKey.currentState.showSnackBar(const SnackBar(
+                        ScaffoldMessenger.of(context).showSnackBar(const SnackBar(
```

Comment:
Should you also get rid of _scaffoldKey itself now?  Same goes for other files with a _scaffoldKey, if it doesn't serve some other purpose.

Reply:
Ah, Yes thank you! Will do!